_episodes_rmd/01-introduction.Rmd

```diff
@@ -177,32 +180,30 @@ convention, R scripts end with the file extension **.R**.
 
 ## Overview and customization of the RStudio layout
 
-Now that we have covered the basics, lets address some ways to configure the
-layout of RStudio. First, here are the major windows (or panes) of the RStudio
+Here are the major windows (or panes) of the RStudio
 environment:
 
 <img src="../fig/rstudio_session_4pane_layout.png" alt="rstudio default session" style="width: 1000px;"/>
 
 - **Source**: This pane is where you will write/view R scripts. Some outputs
-  (such as if you view a dataset using `View()`) will appear as a tab here
+  (such as if you view a dataset using `View()`) will appear as a tab here.
 - **Console/Terminal**: This is actually where you see the execution of
   commands. This is the same display you would see if you were using R at the
   command line without RStudio. You can work interactively (i.e. enter R
   commands here), but for the most part we will run a script (or lines in a
   script) in the source pane and watch their execution and output here. The
   "Terminal" tab give you access to the BASH terminal (the Linux operating
-  system, unrelated to R)
+  system, unrelated to R) - we will not be using this tab.
```

Comment:
We might, though... I could see a workshop instructor wanting to do something in terminal and if they use this there's less context switching?

Reply:
removed that phrase